CHANGELOG.md

```diff
@@ -119,5 +119,67 @@ This option is new for VectorCAST 23 sp2
 - Added support for relative paths to the coded test file in test script #106
 - Fixed issue with unrecoverable link error after a coded test compile error #128
 
+## [1.0.11] - 2024-06-27
 
+### Bug Fixes
+- Removed another instance where the VCAST version mismatch error was displayed in the output pane.
```

Comment:
VectorCAST :)

Reply:
Changed in dfd62cc15233effafc5d3bb680e252d54156de5c.